src/video-player/components/runtime.scss

```diff
@@ -0,0 +1,43 @@
+@import "../../shared/styles/helpers";
+
+.runtime {
+  @include lara-styles;
+}
```

Comment:
If you refactor this interactive to use `BaseQuestionApp`, these lines won't be necessary. Actually, they'll need to be removed, as otherwise the default padding will be added twice.